src/test/url-loader/redirect-resolver_test.ts

```diff
@@ -25,12 +25,12 @@ suite('RedirectResolver', function() {
           new RedirectResolver(resolvedUrl``, 'proto://site/', 'some/path/');
       assert.equal(
           resolver.resolve(
-              fileRelativeUrl`proto://site/something.html`, resolvedUrl``),
+              resolvedUrl``, fileRelativeUrl`proto://site/something.html`),
```

Comment:
Can just make these calls the one-argument packageRelativeUrl form. I think it's clearer and more closely aligned with the intent of the tests

Reply:
Good point.  Done.